harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -41,7 +41,7 @@ def test_executing_eagerly():
 )
 def xor_trial_controller(request):
     """
-    This fixture will provide a function that takes a hyperparameters dictionary as input and
+    This fixture provides a function that takes a hyperparameter's dictionary as input and
```

Comment:
pretty sure the apostrophe is wrong.  `hyperparameter dictionary` might be better, but `hyperparameter's dictionary` is definitely not right.

Reply:
Ack, you're right. The meaning is "a dictionary of hyperparameters".